addons/addon-base-raas/packages/base-raas-cfn-templates/src/templates/service-catalog/ec2-linux-instance.cfn.yml

```diff
@@ -133,6 +136,9 @@ Resources:
           FromPort: -1
           ToPort: -1
           CidrIp: !Ref AccessFromCIDRBlock
+        - DestinationSecurityGroupId:
+            Fn::ImportValue: !Sub "${SolutionNamespace}-CfnEndpointSecurityGroup"
+          IpProtocol: '-1'
```

Comment:
Don't we need isAppStream flag? Does this work without that flag?

Reply:
Good catch. Adding the flag.